packages/react-async/src/tests/Async.test.tsx

```diff
@@ -94,11 +112,43 @@ describe('<Async />', () => {
     expect(renderLoading).toHaveBeenCalled();
     expect(async).toContainReact(<Loading />);
   });
+
+  describe('defer', () => {
+    it('loads the module in an idle callback when defer is DeferTiming.Idle', () => {
+      const promise = createResolvablePromise({default: MockComponent});
+      const load = jest.fn(() => promise.promise);
+
+      mount(<Async defer={DeferTiming.Idle} load={load} render={() => null} />);
+      expect(load).not.toHaveBeenCalled();
+
+      requestIdleCallback.runIdleCallbacks();
+      expect(load).toHaveBeenCalled();
+    });
+
+    it('loads the module in on viewport intersection when defer is DeferTiming.InViewport', () => {
```

Comment:
`loads the module in on viewport intersection` => `loads the module on viewport intersection `